cluster/gce/gci/configure-helper.sh

```diff
@@ -208,8 +208,9 @@ function create-master-auth {
   local -r known_tokens_csv="${auth_dir}/known_tokens.csv"
   if [[ ! -e "${known_tokens_csv}" ]]; then
     echo "${KUBE_BEARER_TOKEN},admin,admin" > "${known_tokens_csv}"
-    echo "${KUBELET_TOKEN},kubelet,kubelet" >> "${known_tokens_csv}"
-    echo "${KUBE_PROXY_TOKEN},kube_proxy,kube_proxy" >> "${known_tokens_csv}"
+    echo "${KUBE_CONTROLLER_MANAGER_TOKEN}kcmtoken,system:kube-controller-manager,uid:system:kube-controller-manager" >> "${known_tokens_csv}"
```

Comment:
@cjcullen I think this token value is coming through empty even though I tried to set it in the `common.sh` file.  Can you help figure out why?